### PR TITLE
additional mimetypes, fix #380

### LINKED ISSUE #380: bashlib: more image mimetypes
The following are missing in `ocrd_models.constants.EXT_TO_MIME` and `MIME_TO_EXT`:

- image/jp2
- image/pdf
- image/ps
- image/eps
- image/xps
- image/ppm
- image/pnm
- image/pbm

I know these are not meant to be processed in OCR-D. But think ocrd-im6convert etc.

### PR DESCRIPTION
**Pull request by kba**

### CODE REVIEW

File: ocrd_utils/ocrd_utils/constants.py [hunk 1]
```diff
@@ -20,7 +20,15 @@
     '.png': 'image/png',
     '.jpg': 'image/jpg',
     '.jpeg': 'image/jpg',
-    '.xml': MIMETYPE_PAGE
+    '.xml': MIMETYPE_PAGE,
+    '.jp2': 'image/jp2',
+    '.pdf': 'image/pdf',
+    '.ps': 'image/ps',
+    '.eps': 'image/eps',
+    '.xps': 'image/xps',
```
Comment (bertsky) [inline ocrd_utils/ocrd_utils/constants.py]:
```suggestion
    '.pdf': 'application/pdf',
    '.ps': 'application/postscript',
    '.eps': 'application/postscript',
    '.xps': 'application/oxps',
```

File: ocrd_utils/ocrd_utils/constants.py [hunk 2]
```diff
@@ -30,4 +38,12 @@
     'image/jpeg': '.jpg',
     MIMETYPE_PAGE: '.xml',
     'application/alto+xml': '.xml',
+    'image/jp2': '.jp2',
+    'image/pdf': '.pdf',
+    'image/ps': '.ps',
+    'image/eps': '.eps',
+    'image/xps': '.xps',
```
Comment (bertsky) [inline ocrd_utils/ocrd_utils/constants.py]:
```suggestion
    'application/pdf': '.pdf',
    'application/postscript': '.eps',
    'application/oxps': '.xps',
```

File: ocrd_utils/ocrd_utils/constants.py [hunk 3]
```diff
@@ -20,7 +20,15 @@
     '.png': 'image/png',
     '.jpg': 'image/jpg',
     '.jpeg': 'image/jpg',
```
Comment (bertsky) [inline ocrd_utils/ocrd_utils/constants.py]:
```suggestion
    '.jpg': 'image/jpeg',
    '.jpeg': 'image/jpeg',
```

**Comment by kba** [comment]
> Some of the MIME types were invalid.

I'll fix them, thanks.

**Pull request merged**